test/README.md

```diff
@@ -0,0 +1,19 @@
+Linuxkit Tests Labels
+=====================
+
+## Usage of Artifacts vs Temporary Directory
+
+As the Build Machines have no secrets they will not be able to test running the build output on any cloud providers.
+In this instance, the build tests should copy their output to the `LINUXKIT_ARTIFACTS_DIRECTORY`
```

Comment:
This is referring to the fact that we have unprivileged disposable GCP VM's that run (pretty much) all of the tests... and some Python code in the CI itself that attempts to run the `test.img.tar.gz` produced during the test phase on GCP.

That file gets `scp`'d back from the VM before it's destroyed
As this was a pattern I'd assumed we would continue I tried (and failed) to formalise it in the README.

It's a bit messy though so see below...